pkg/recipes/types.go

```diff
@@ -129,3 +133,11 @@ func (ro *RecipeOutput) PrepareRecipeResponse(resultValue map[string]any) error
 
 	return nil
 }
+
+func (ro *RecipeOutput) AddRecipeStatus(templateKind string, templatePath string, templateVersion string) {
```

Comment:
If this is just going to set fields, do we need a function for it? I don't have a strong feeling about this, but IMO it's simpler if we just set the fields directly.